examples/scripts/compute.py

```diff
@@ -0,0 +1,64 @@
+import os
+import pandas as pd
+from oemof.solph import EnergySystem, Model
+from oemof.solph.processing import parameter_as_dict
+
+# ---- imports to be used when the package has been installed ----
+from oemof.tabular import datapackage # noqa
+from oemof.tabular.facades import TYPEMAP
+# ---- imports from oemof-tabular-plugins package ----
+# ToDo: adapt the way these imports are called once oemof-tabular-plugins has expanded a bit
```

Comment:
if you install the package locally with `pip install -e .` at the root of the repository you can perfectly use import statements like `from oemof_tabular_plugins.general.post_processing import post_processing`